StRoot/StETofCalibMaker/StETofCalibMaker.h

```diff
@@ -136,6 +138,7 @@ class StETofCalibMaker: public StMaker {
     std::string   mFileNameTimingWindow;        // name of parameter file for timing window
     std::string   mFileNameSignalVelocity;      // name of parameter file for signal velocity
     std::string   mFileNameCalibHistograms;     // name of parameter file for calibration histograms (output of QA maker)
+	 std::string   mFileNameOffsetHistograms;    // name of parameter file for run by run offsets histograms (output of QA maker)
```

Comment:
You add tab indented lines to the code that uses whitespaces

Reply:
fixed in latest commit